vmm/src/vm.rs

```diff
@@ -25,7 +25,9 @@ extern crate vm_allocator;
 extern crate vm_memory;
 extern crate vm_virtio;
 
-use crate::config::{DeviceConfig, DiskConfig, HotplugMethod, NetConfig, PmemConfig, VmConfig};
+use crate::config::{
```

Comment:
Typo in commit message: `When performing an API boot validate that the configuration.`
Seems like the sentence is not complete :)